models/swagger/access_key_params.go

```diff
@@ -0,0 +1,16 @@
+package swagger
+
+import "time"
+
+type AccessKeyParams struct {
+	IncludeDeleted *bool `json:"includeDeleted,omitempty" swaggertype:"boolean" default:"false" example:"true"`
+
+	CreatedRangeFrom *time.Time `json:"createdRange[from],omitempty" example:"2024-02-26T11:01:28Z"`
+	CreatedRangeTo   *time.Time `json:"createdRange[to],omitempty" example:"2024-02-26T11:01:28Z"`
+
+	UpdatedRangeFrom *time.Time `json:"updatedRange[from],omitempty" example:"2024-02-26T11:01:28Z"`
+	UpdatedRangeTo   *time.Time `json:"updatedRange[to],omitempty" example:"2024-02-26T11:01:28Z"`
```

Comment:
Those 3 fields are common for each model filter,
- maybe we could have a struct containing those common search params
- maybe the struct with common search params could also contain metadata (instead of having it in a separate struct?